full-node/db/sov-db/src/schema/tables.rs

```diff
@@ -157,7 +196,7 @@ define_table_with_default_codec!(
     (BatchByHash) DbHash => BatchNumber
 );
 
-define_table_with_default_codec!(
+define_table_with_u64_wrapper_keys!(
     /// The primary source for transaction data
     (TxByNumber) TxNumber => StoredTransaction
 );
```

Comment:
It looks like we're still using the default (little endian) serializer for `EventByNumber` and `SlotNumber`

Reply:
Fixed in https://github.com/Sovereign-Labs/sovereign-sdk/pull/443/commits/ae4c3d894b5f6d0609a26b784b3458fde56f34b7. I also renamed `define_table_with_u64_wrapper_keys` to `define_table_with_seek_key_codec` and switched to big-endian `bincode` to make it more general-purpose, in preparation for using it on other types such as `NodeKey`. In the meantime I'm removing the blanket `SeekKeyEncoder` implementation to avoid misuse as much as possible.